tsconfig.json

```diff
@@ -41,15 +41,13 @@
         "resolveJsonModule": true,
         "esModuleInterop": true,
         "allowSyntheticDefaultImports": true,
-        "preserveConstEnums": true,
         "importHelpers": true,
         "noFallthroughCasesInSwitch": true,
         "noUnusedLocals": true,
         "noImplicitReturns": true,
         "noUnusedParameters": true,
         "incremental": true,
         "newLine": "lf",
-        "isolatedModules": false,
```

Comment:
Option 'preserveConstEnums' cannot be disabled when 'isolatedModules' is enabled.